app/workers/certificates.php

```diff
@@ -125,6 +125,9 @@ public function run(): void
             $attempts = $certificate->getAttribute('attempts', 0) + 1;
             $certificate->setAttribute('attempts', $attempts);
 
+            // Store cuttent time as renew date to ensure another attempt in next maintenance cycle
+            $certificate->setAttribute('renewDate', DateTime::now());
```

Comment:
I think so.  In maintanance there is query that does:
```php
[
               Query::lessThan('attempts', 5),
               Query::lessThanEqual('renewDate', $time),
               Query::limit(200),
]
```

If first attempt fails, renewDate stays empty and never hits retry loop.
Tho it worked fine for when certificate renewing fails.

Dont think it's too critical bug, if you are considering 0.15.4 because of this.